packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -949,6 +979,22 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
     }
   }
 
+  /// Selects a position at the `offset` location.
+  ///
+  /// The selection will be collapsed at the `offset` location.
+  ///
+  /// See also:
+  ///  * [_selectStartTo], which sets or updates selection start edge.
+  ///  * [_selectEndTo], which sets or updates selection end edge.
+  ///  * [_finalizeSelection], which stops the `continuous` updates.
+  ///  * [_clearSelection], which clear the ongoing selection.
+  ///  * [_selectWordAt], which selects a whole word at the location.
+  ///  * [selectAll], which selects the entire content.
+  void _selectPositionAt({required Offset offset}) {
```

Comment:
The name sounds to me like it will not result in a collapsed selection.  Maybe collapseSelectionAt or something like that?